tmate-ssh-server.c

```diff
@@ -208,7 +208,10 @@ static void handle_sigalrm(__unused int sig)
 	tmate_fatal_quiet("Connection grace period (%ds) passed", TMATE_SSH_GRACE_PERIOD);
 }
 
-static void client_bootstrap(struct tmate_session *_session)
+static void client_bootstrap(struct tmate_session *_session,
+                             bool handle_role_daemon,
+                             bool handle_role_pty_client,
+                             bool handle_role_exec)
```

Comment:
I believe you only need 2 boolean: you handle clients or servers (or both, although it is not the case here). If you handle clients, you will want to handle pty *and* exec requests alike (I don't think we want to have separate policies for the two).